src/main/java/frc/robot/RobotContainer.java

```diff
@@ -128,14 +130,15 @@ public void putShuffleboardCommands() {
             SmartShuffleboard.putCommand("Climber", "Climb", new StaticClimb(climber));
         }
         if (Constants.INTAKE_DEBUG){
-        SmartShuffleboard.putCommand("Intake", "Start Intake", new StartIntake(intakeSubsystem));
+        SmartShuffleboard.putCommand("Intake", "Start Intake", new StartIntake(intakeSubsystem,5));
         }
     }
 
     private void configureBindings() {
         drivetrain.setDefaultCommand(new Drive(drivetrain, joyleft::getY, joyleft::getX, joyright::getX));
         joyLeftButton1.onTrue(new InstantCommand(() -> drivetrain.setAlignable(Alignable.SPEAKER))).onFalse(new InstantCommand(()-> drivetrain.setAlignable(null)));
         joyRightButton1.onTrue(new InstantCommand(() -> drivetrain.setAlignable(Alignable.AMP))).onFalse(new InstantCommand(()-> drivetrain.setAlignable(null)));
+        intakeSubsystem.setDefaultCommand(new SpinIntake(intakeSubsystem, controller::getLeftY));
```

Comment:
I don't expect we'll want this for real but I haven't figured out a good means of defining interim vs final joystick bindings

Reply:
got rid of defualt command